server/webapp/WEB-INF/rails.new/app/assets/javascripts/console_log_foldable_section.js

```diff
@@ -64,37 +70,80 @@
       }
     }
 
-    function insertBasic(cursor, line) {
-      var output = c("dd", {class: "log-fs-line"}, ansi.ansi_to(line, formatter));
+    function lpad3(num) {
+      return ("00" + num).slice(-3);
+    }
+
+    // no, moment.js doesn't make this easy (at least not without some plugin)
+    function humanizeMilliseconds(duration) {
```

Comment:
From a maintenance point-of-view, would much rather prefer to use [moment-duration-format](https://github.com/jsmreese/moment-duration-format).

It seems I can do this, which feels much nicer


```
var x = moment.duration(10000000123, 'ms')
console.log(x.format('d [days] h:m:s.S'))
=> 115 days 17:46:40.123
```

Might mean for us to do something like this, or something better?

```
d = moment.duration(someDuration, 'ms')
if (d <= 1 second) {
  return d.format('s.S [sec]')
} else if (d <= 1 minute) {
  return d.format('m [min] s.S [sec]')
} else if (d <= 1 hour) {
  ...
}
...
```